<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jan 'Kovi≈°' Struhar</author>
 <class>ReportColumnConfigurationDlg</class>
 <widget class="QDialog" name="ReportColumnConfigurationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Column selection</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Columns to be visible in reports</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="m_visibleColumns">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Use move left &lt;img src=&quot;:/images/1leftarrow.png&quot; /&gt;/move right &lt;img src=&quot;:/images/1rightarrow.png&quot; /&gt;buttons or drag the columns to change their order. Drag&amp;amp;drop the columns to the other table to hide them - or use hide &lt;img src=&quot;:/images/1downarrow.png&quot; /&gt; button</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectColumns</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="columnLeftButton">
     <property name="text">
      <string>Move column to the left</string>
     </property>
     <property name="icon">
      <iconset resource="HolyServices.qrc">
       <normaloff>:/images/1leftarrow.png</normaloff>:/images/1leftarrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="columnRightButton">
     <property name="text">
      <string>Move column to the right</string>
     </property>
     <property name="icon">
      <iconset resource="HolyServices.qrc">
       <normaloff>:/images/1rightarrow.png</normaloff>:/images/1rightarrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="hideColumnButton">
     <property name="text">
      <string>Hide column</string>
     </property>
     <property name="icon">
      <iconset resource="HolyServices.qrc">
       <normaloff>:/images/1downarrow.png</normaloff>:/images/1downarrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Columns that will be NOT visible in reports</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QTableView" name="m_hiddenColumns">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Press &quot;Expose&quot; &lt;img src=&quot;:/images/1uparrow.png&quot; /&gt; button or drag it to the other table to make it visible in report</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectColumns</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QPushButton" name="exposeColumnButton">
     <property name="toolTip">
      <string>Moves the selected column to the other table</string>
     </property>
     <property name="text">
      <string>Make the column appear in reports</string>
     </property>
     <property name="icon">
      <iconset resource="HolyServices.qrc">
       <normaloff>:/images/1uparrow.png</normaloff>:/images/1uparrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="HolyServices.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportColumnConfigurationDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportColumnConfigurationDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
